site/docs/v1/tech/identity-providers/google.adoc

```diff
@@ -112,21 +112,29 @@ The Google Client Secret found in your Google credentials settings in the `Clien
 
 [field]#Button text# [required]#Required#::
 The text to be displayed in the button on the login form. This value is defaulted to `Login with Google` but it may be modified to your preference.
++
+[since]#Since 1.44.0#
+The [field]#Button text# value will not be used when [field]#Login method# is set to `Use popup for login` because of changes to the Google JavaScript API.
 
 [field]#Login Method# [optional]#Optional#::
 User interaction behavior after clicking on the IdP button (`Login with Google`, for instance).
 +
-* `Use redirect for login` - if selected, the user is redirected to a Google login page. Once authenticated, the user is redirected back to FusionAuth.
+ * `Use redirect for login` - if selected, the user is redirected to a Google login page. Once authenticated, the user is redirected back to FusionAuth.
 If selected, the redirect URL *must* be set to an absolute URL in the Google console for your application. If your hostname is login.piedpiper.com, the redirect URL would be `https://login.piedpiper.com/oauth2/callback/`.
-+
-* `Use popup for login` - if selected, a popup is displayed to the user to login with google. Once authenticated, the window is closed.
+ * `Use popup for login` - if selected, a popup is displayed to the user to login with google. Once authenticated, the window is closed.
 If selected, the `Authorized JavaScript origins` URL *must* be allowed for your host name in the Google console for your application. For example, `\https://login.piedpiper.com`.
 +
 Please note if an `idp_hint` is appended to the OAuth Authorize endpoint, then the interaction behavior will be `redirect`, even if popup interaction is explicitly configured.
 +
+[since]#Since 1.44.0#
+The `Use popup for login` method will render the button in an `iframe` because of changes to the Google JavaScript API.
```

Comment:
```suggestion
The `Use popup for login` method will render the button in an `iframe` because of changes to the Google JavaScript API.

If you are using a version of FusionAuth < 1.44.0, `Use popup for login` won't work after Mar 31 2023. Please see (Issue #1939)[https://github.com/FusionAuth/fusionauth-issues/issues/1939] for more.
```